### ISSUE TITLE
Passing in `backgroundColor` as a style object property for a `Text` slot is not respected in HC

### ISSUE THREAD

**Issue by rurikoaraki** [microsoft/fluentui-react-native#1394]
### Willing to submit a PR to fix?

- [X] I am willing to submit a PR to fix

### Requested priority

Normal

### Products/applications affected

Button, Text

### Package version(s)

latest master, using Metro

### OS version(s)

10.0.22000.434

### Platform

- [ ] iOS
- [ ] macOS
- [ ] win32
- [X] windows
- [ ] android

### Please provide a reproduction of the bug

1. Add "backgroundColor: 'red'," to line 98 of Button.styling.ts
2. Turn on HC in Windows
3. Load the Windows app
4. Observe left nav pane

### Actual behavior

Text of Button is not visible

### Expected behavior

Text of Button is visible
